server/lib/src/idm/scim.rs

```diff
@@ -546,15 +546,7 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
         &mut self,
         sse: &'b ScimSyncUpdateEvent,
         changes: &'b ScimSyncRequest,
-    ) -> Result<
-        (
-            Uuid,
-            BTreeSet<String>,
-            BTreeMap<Uuid, &'b ScimEntryGeneric>,
-            bool,
-        ),
-        OperationError,
-    > {
+    ) -> Result<(Uuid, BTreeSet<String>, BTreeMap<Uuid, &'b ScimEntry>, bool), OperationError> {
```

Comment:
can we make the return a struct so it's clearer to folks what is coming back? I know it's in your head now but that's a ... lot of stuff 😄